settings.gradle

```diff
@@ -1 +1 @@
-include ':core', ':keycloak-service-module', ':auth'
```

Comment:
I'm not sure if we want to have example as top level gradle project or just be completely standalone - which will more or less simulate how end users will use this library

Current aproach looks ok for people to try this out and it's ok but raising to get other opinions.

Reply:
I can't see any problem to have the sample here. Most of the libraries do the same

See: https://github.com/facebook/facebook-android-sdk/blob/master/settings.gradle

Anyway, I have no problem to remove it, if you want!

Reply:
I can't see any problem to have the sample here. Most of the libraries do the same

See: https://github.com/facebook/facebook-android-sdk/blob/master/settings.gradle

Anyway, I have no problem to remove it, if you want!

Reply:
@danielpassos @wtrocki I think having it separate would make users' lives easier. They would just clone the example app and modify it.

Reply:
I do not have strong opinion about that. Just wanted to flag that and see other opinions. IMHO we should merge that PR as it will block Ali documentation work

Reply:
@aliok @wtrocki @danielpassos hmm, on the other hand i always like it when libraries have an `example` directory. Isn't @secondsun also working on a separate kitchensink app?

Reply:
@pb82 This was just initial app for demo purposes. This one may be be more or less documentation how to use libraries and it should be very simple

Reply:
@aliok This is what we call cookbook or template, we are also going to create a separated one for that, this is more for us playing with features.

Reply:
sounds good :)